README.md

```diff
@@ -156,6 +156,46 @@ Run a deployment you must set env var `IMAGE_TAG`
 skaffold run
 ```
 
+## Add new application property
+1. Locally, application properties are controlled by the values in the various application.properties files.
+    - `application.properties` contains the base/default values. A new property should be added here first.
+    ```
+    datarepo.gcs.allowReuseExistingBuckets=false
+    ```
+    - You can override the default value for connected and integration tests by adding a line to 
+    `application-connectedtest.properties` and `application-integrationtest.properties`.
+    ```
+    datarepo.gcs.allowReuseExistingBuckets=true
+    ```
+2. Now that we use Helm, the properties also need to be added to the
+[base Data Repo charts](https://github.com/broadinstitute/datarepo-helm).
+    - Find the the [api-deployment.yaml](https://github.com/broadinstitute/datarepo-helm/blob/master/charts/datarepo-api/templates/api-deployment.yaml) file.
+    - Add a new property under the `env` section. The formatting below might be messed up, and the yaml is very picky
+    about spaces. So, copy/paste from another variable in the section instead of here.
+    ```
+            {{- if .Values.env.datarepoGcsAllowreuseexistingbuckets }}
+            - name: DATAREPO_GCS_ALLOWREUSEEXISTINGBUCKETS
+              value: {{ .Values.env.datarepoGcsAllowreuseexistingbuckets | quote }}
+            {{- end }}
+    ```
+    - Find the the [values.yaml](https://github.com/broadinstitute/datarepo-helm/blob/master/charts/datarepo-api/values.yaml) file.
+    - Add a new line under the `env` section.
+    ```
+      datarepoGcsAllowreuseexistingbuckets:
+    ```
+    - Release a new version of the chart. Talk to DevOps/Mark S to do this.
```

Comment:
```suggestion
    - Release a new version of the chart. Talk to DevOps to do this.
```

Reply:
Based on conversations I had with Jeremy for the getting started guide, we've avoided calling out specific people.

Reply:
Done, thanks.